v2/components/schemas/Permissions.yaml

```
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+required:
+- outcome
+- lifespan
+properties:
+  outcome:
+    type: string
+    enum:
+      - allow
+      - deny
+  lifespan:
+    type: string
+    enum:
+      - timespan
+      - session
+      - forever
+  expiration:
+    type: string
+    description: Only occurs if lifespan is set to 'timespan'.
```

I think this should include a description of what the expiration is

```suggestion
    description: |-
    The timestamp at which the permission will expire.    

    Only used if lifespan is 'timespan'.
```